client/driver/docker.go

```diff
@@ -17,12 +17,13 @@ import (
 	"time"
 
 	"github.com/armon/circbuf"
-	docker "github.com/fsouza/go-dockerclient"
+	"github.com/fsouza/go-dockerclient"
```

Comment:
Goimports suggested this change. Please let me know if you want me to rollback this line.

Reply:
its because the imported package in that folder is already called `docker` :)